management.md

```diff
@@ -32,6 +32,38 @@
 - What is your team's software development and testing process for creating
   new features?
 
+  - Utilize the Issue Tracker
+
+    When developing a new feature the first step is to create an issue in the
+    GitHub Issue Tracker. This issue should outline the purpose of the new
+    feature and the steps needed for feature development. If this feature solves
+    any current issues open in the Issue Tracker, connect the issue numbers.
+    Next, assign the members of your development team and request feedback
+    from team members, Technical Leaders, and the course instructor.
+
+  - Create a Branch
+
+    The next step is to create a branch off of master with the following naming
+    convention: feature/description-of-the-feature. All the work for the new
```

Comment:
Small issue: I would add "`" around feature/description-of-the-feature

Reply:
Got it! Thanks!

Reply:
I actually meant to use the backtick, instead of the quotation mark, sorry for the confusion